linty_fresh/main.py

```diff
@@ -1,7 +1,12 @@
 import argparse
 import asyncio
 
-from linty_fresh.linters import android, checkstyle, mypy, pylint, swiftlint
+from linty_fresh.linters import android
```

Comment:
yeah, the line was too long for the linter so I split it to single line imports.  Its actually slightly preferred but you can do it either way.